src/Service/S3/src/Signer/SignerV4ForS3.php

```diff
@@ -36,6 +36,15 @@ class SignerV4ForS3 extends SignerV4
         'PutObjectLockConfiguration' => true,
     ];
 
+    private $sendChunkedBody;
+
+    public function __construct(string $scopeName, string $region, bool $sendChunkedBody = true)
```

Comment:
Does it make sense to pass an array with options here instead? Or is it just me that is defensive?

Reply:
updated